contracts/shapella-upgrade/ShapellaUpgradeTemplate.sol

```diff
@@ -0,0 +1,1096 @@
+     * @dev DEPRECATED: Now fees information is stored in StakingRouter and
+     * with higher precision. Use StakingRouter.getStakingFeeAggregateDistribution() instead.
+     * @return treasuryFeeBasisPoints return treasury fee in TOTAL_BASIS_POINTS (10000 is 100% fee) precision
+     * @return insuranceFeeBasisPoints always returns 0 because the capability to send fees to
+     * insurance from Lido contract is removed.
+     * @return operatorsFeeBasisPoints return total fee for all operators of all staking modules in
+     * TOTAL_BASIS_POINTS (10000 is 100% fee) precision.
+     * Previously returned total fee of all node operators of NodeOperatorsRegistry (Curated staking module now)
+     * The value might be inaccurate because the actual value is truncated here to 1e4 precision.
+     */
+    function getFeeDistribution() external view
+        returns (uint16 treasuryFeeBasisPoints, uint16 insuranceFeeBasisPoints, uint16 operatorsFeeBasisPoints);
+
+    /**
+     * @notice Returns current staking rewards fee rate
+     * @dev DEPRECATED: Now fees information is stored in StakingRouter and
+     * with higher precision. Use StakingRouter.getStakingFeeAggregateDistribution() instead.
+     * @return totalFee total rewards fee in 1e4 precision (10000 is 100%). The value might be
+     * inaccurate because the actual value is truncated here to 1e4 precision.
+     */
+    function getFee() external view returns (uint16 totalFee);
+}
+
+interface ILidoLocator is IOssifiableProxy {
+    function accountingOracle() external view returns(address);
+    function depositSecurityModule() external view returns(address);
+    function elRewardsVault() external view returns(address);
+    function legacyOracle() external view returns(address);
+    function lido() external view returns(address);
+    function oracleReportSanityChecker() external view returns(address);
+    function burner() external view returns(address);
+    function stakingRouter() external view returns(address);
+    function treasury() external view returns(address);
+    function validatorsExitBusOracle() external view returns(address);
+    function withdrawalQueue() external view returns(address);
+    function withdrawalVault() external view returns(address);
+    function postTokenRebaseReceiver() external view returns(address);
+    function oracleDaemonConfig() external view returns(address);
+}
+
+interface ILegacyOracle is IVersioned {
+    /**
+     * @notice A function to finalize upgrade v3 -> v4 (the compat-only deprecated impl).
+     * Can be called only once.
+     */
+    function finalizeUpgrade_v4(address accountingOracle) external;
+}
+
+interface ILidoOracle {
+    /**
+     * @notice Return the initialized version of this contract starting from 0
+     */
+    function getVersion() external view returns (uint256);
+
+    /**
+     * @notice Return the current oracle member committee list
```

Comment:
maybe committee member list

Reply:
fixed, now the doc is better than it is on-chain)

Reply:
Sorry, can't see the change yet